--- a/plugins/ftdidmx/FtdiDmxThread.cpp	2019-03-03 11:12:28.000000000 -0800
+++ b/plugins/ftdidmx/FtdiDmxThread.cpp	2019-03-03 11:12:50.000000000 -0800
@@ -111,7 +111,7 @@
     }
 
     if (m_granularity == GOOD) {
-      usleep(DMX_BREAK);
+      usleep(DMX_BREAK*4);
     }
 
     if (!m_interface->SetBreak(false)) {
@@ -119,7 +119,7 @@
     }
 
     if (m_granularity == GOOD) {
-      usleep(DMX_MAB);
+      usleep(DMX_MAB*2);
     }
 
     if (!m_interface->Write(buffer)) {
