--- a/plugins/ftdidmx/FtdiDmxPlugin.cpp	2019-03-02 23:12:45.000000000 -0800
+++ b/plugins/ftdidmx/FtdiDmxPlugin.cpp	2019-03-02 23:12:55.000000000 -0800
@@ -125,7 +125,7 @@
   }
 
   if (m_preferences->SetDefaultValue(FtdiDmxPlugin::K_FREQUENCY,
-                                     UIntValidator(1, 44),
+                                     UIntValidator(1, 440),
                                      DEFAULT_FREQUENCY)) {
     m_preferences->Save();
   }
